v3/examples/mobile/frontend/public/style.css

```
@@ -0,0 +1,149 @@
+    margin-bottom: 14px;
+}
+
+.output {
+    background: #0f1726;
+    color: #c8d6ec;
+    border-radius: var(--radius);
+    padding: 12px 14px;
+    font-size: 13px;
+    line-height: 1.5;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)